pkg/server/routes.go

```diff
@@ -539,4 +539,13 @@ func (s *Server) routes(
 
 	api.Handle("/pdf/generate", handlers.NewPDFHandler(services.NewInvokeGeneratePDF(serviceConfig, lambdaClient, princeLambdaName)).Handle())
 
+	systemsHandler := handlers.NewSystemsHandler(
```

Comment:
Seeing as this handler returns fake data, can we feature flag it so Prod can't access it?

Reply:
Yupperz! I plan to feature flag real vs fake data in a later PR.